scripts/linux/run_client.sh

```diff
@@ -1,4 +1,20 @@
-BOOTSPR=UPDATE ME
+#!/bin/env bash
+set -e
+
+if [ -z "$LOCALIP" ]; then
+  echo "LOCALIP is not defined. Please set it to your IP address."
+  exit 1
+fi
+
+if [ -z "$BOOTSPR" ]; then
+  echo "BOOTSPR is not defined. Please set it to the bootstrap node SPR."
```

Comment:
How are attendees going to copy an SPR into the command line? 

Possibly, what we should do here is `BOOTSPR` is not defined, we can fallback to a default one, which we can update to be the local network's bootstrap SPR. For the All Codex dry run, we can set it to one of the bootstrap node's SPRs on the testnet